app/graphql/types/customers/update_customer_input.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Types
+  module Customers
+    class UpdateCustomerInput < BaseInputObject
+      description 'Update Customer input arguments'
+
+      argument :address_line1, String, required: false
+      argument :address_line2, String, required: false
+      argument :city, String, required: false
+      argument :country, Types::CountryCodeEnum, required: false
+      argument :currency, Types::CurrencyEnum, required: false
+      argument :email, String, required: false
+      argument :external_id, String, required: true
+      argument :id, ID, required: true
```

Comment:
What do you think of adding this ID field as a first argument separated from the other by a blank line (to make rubocop happy)?

Reply:
Yeah absolutely! Just pushed an update for it